README.md

```diff
@@ -15,6 +15,7 @@
 <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
 <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
 <a href="https://www.w3schools.com/python/"><img src= "https://img.icons8.com/color/48/000000/python.png"/><a>
+<a href="https://www.tutorialspoint.com/yaml/index.htm"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/48px-Official_YAML_Logo.svg.png"/><a>
```

Comment:
Add the YAML logo after the JS icon @zmusaddique

Reply:
OK

Reply:
![Uploading image.png…]()

Are you happy with this?

Reply:
![image](https://user-images.githubusercontent.com/84737749/198563824-6b994264-a1b6-4307-8d80-40fa8f983656.png)

Is this fine?